kin-sdk-core/src/main/java/kin/sdk/core/EthClientWrapper.java

```diff
@@ -108,15 +100,35 @@ private void initKeyStore(android.content.Context context) throws EthereumClient
         keyStore = Geth.newKeyStore(keystoreDir.getAbsolutePath(), Geth.LightScryptN, Geth.LightScryptP);
     }
 
+    private StringBuilder getKeyStoreBasePath() {
+        return new StringBuilder(androidContext.getFilesDir().getAbsolutePath())
+            .append(File.separator)
+            .append("kin")
+            .append(File.separator)
+            .append("keystore");
+    }
+
+    public String getKeyStoreFullPath() {
+        return getKeyStoreBasePath().append(File.separator)
+            .append(serviceProvider.getNetworkId())
```

Comment:
why do we need. both getKeyStoreFullPath and getKeyStoreBasePath? we can just whip the account in current client network id